### PR TITLE
Gazebo logs

### PR DESCRIPTION
**Pull request by david-shearon**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ mavsdk>=1.4.0
 numpy>=1.23.0
 pymavlink>=2.4.37
 grpcio>=1.47.0
-pyserial>=3.5
\ No newline at end of file
+pyserial>=3.5
+pyulog
```
Comment (justincdavis) [inline requirements.txt:6]: Where do we use this?

File: scripts/tools/gazebo_logger.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+elif(len(sys.argv) == 2):
+    entry = entries[int(sys.argv[1])]
+
+    filename = "log_" + str(sys.argv[1])
+        
+    #Download
+    try:
+        drone.log_files.download_log_file(entry, str(DIR + "RAW/" + filename + ".txt"), 4000)   
+    except: #cleanup
```
Comment (justincdavis) [inline scripts/tools/gazebo_logger.py:53]:
```suggestion
    except Exception: #cleanup
```

File: scripts/tools/gazebo_logger.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+if(len(sys.argv) == 1):
+    #Download and convert all logs to CSV
+    for count in range(len(entries)):
+        filename = "log_" + str(count)
+        
+        #Download
+        try:
+            drone.log_files.download_log_file(entries[count], str(DIR + "RAW/" + filename + ".txt"), 4000)   
+        except: #cleanup
```
Comment (justincdavis) [inline scripts/tools/gazebo_logger.py:37]:
```suggestion
        except Exception: #cleanup
```

**Review by justincdavis — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (david-shearon) [reply requirements.txt:6]: The pyulog is what we need for Ulog to CSV conversions. I forgot about pyserial.

**Review by justincdavis — APPROVED** [review]

**Pull request merged**